Doc/library/select.rst

```diff
@@ -317,11 +317,17 @@ Edge and Level Trigger Polling (epoll) Objects
    +-------------------------+-----------------------------------------------+
    | :const:`EPOLLMSG`       | Ignored.                                      |
    +-------------------------+-----------------------------------------------+
+   | :const:`EPOLLWAKEUP`    | Prevents sleep during event waiting.          |
```

Comment:
Alternatively explicitly silence them with `!`. I'm fine with either.

Is it possible to use `.. const::` inside the table? That could also work.

Reply:
> Is it possible to use .. const:: inside the table? That could also work.

I... don't know. Never tried to but I wouldn't expect it to work because you'll need to know what the body of the directive is. (but maybe it works without me knowing this)

Reply:
> Depending on what we land on, the `.. versionadded` text and possibly the NEWS entry may need amendments.

It used to be `3.14`, do I need to revert it?

Reply:
> Alternatively explicitly silence them with `!`. I'm fine with either.
> 
> Is it possible to use `.. const::` inside the table? That could also work.

In my impression, `.. xxx::` is usually considered as a paragraph, but maybe it works, e.g. `.. availability:: Unix` or `.. versionadded:: xxx`, and so on. So, maybe `.. const:: xxx` is ok.

Reply:
/me:
> Is it possible to use `.. const::` inside the table? That could also work.

Using `.. data:: NAME` works (inside the table), but it renders too verbose IMO. Let's not do anything with this for now. We can follow up with a doc style amendment later.

RUANG:
> It used to be 3.14, do I need to revert it?

No, I was talking about the Sphinx markup only.

Reply:
> Is it possible to use .. const:: inside the table? That could also work.

It is; we use it [here](https://docs.python.org/3/c-api/structures.html#member-types) for example.

Reply:
How dare it works without me knowing it!